### PR TITLE
Change supplemental water, etc.

### PR DESCRIPTION
**Pull request by XX-Yin**
### Pull Request instructions:
- Please follow the [update protocol](https://github.com/AllenNeuralDynamics/aind-behavior-blog/wiki/Software-Update-Procedures)
- Answer the questions below in detail. Your responses will be emailed to experimenters. 
- If the experimenters must do anything new, provide detailed step by step instructions

### Describe changes:
1) Removing animal name [#62](https://github.com/AllenNeuralDynamics/dynamic-foraging-task/issues/62)
2) Updating the calculation of water suggestion [discussion #64](https://github.com/AllenNeuralDynamics/aind-behavior-blog/discussions/64)
3) Showing the newscale position of the last trial when loading a json file. Increasing the speed of newscale stage. [discussion #80](https://github.com/AllenNeuralDynamics/aind-behavior-blog/discussions/80#discussioncomment-7635249) 

### What issues or discussions does this update address?
 [#62](https://github.com/AllenNeuralDynamics/dynamic-foraging-task/issues/62)
 [discussion #64](https://github.com/AllenNeuralDynamics/aind-behavior-blog/discussions/64)
 [discussion #80](https://github.com/AllenNeuralDynamics/aind-behavior-blog/discussions/80#discussioncomment-7635249) 
### Describe the expected change in behavior from the perspective of the experimenter
Pay attention to the change of water suggestion. We use new rule to calculate the water suggestion [discussion 111](https://github.com/AllenNeuralDynamics/aind-behavior-blog/issues/111#issuecomment-1830812176). 

### Describe the outcome of testing this update on a rig in 447
Tested in Green1 and Green2 over 2 days. No obvious issues observed. 

### Should the experimenters be on the look out for any new behavior?
Pay attention to the change of water suggestion. We use new rule to calculate the water suggestion [discussion 111](https://github.com/AllenNeuralDynamics/aind-behavior-blog/issues/111#issuecomment-1830812176).

### CODE REVIEW

**Comment by alexpiet** [comment]
Based on the SOP, merge updates into `production_testing` not `main`

File: src/foraging_gui/Foraging.py [hunk 1]
```diff
@@ -2049,7 +2052,7 @@ def _Timer(self,Time):
     
     def _set_metadata_enabled(self, enable: bool):
         '''Enable or disable metadata fields'''
-        self.AnimalName.setEnabled(enable)
+        #self.AnimalName.setEnabled(enable)
```
Comment (alexpiet) [inline src/foraging_gui/Foraging.py:2055]: instead of commenting out this line, can you delete it?

Comment (XX-Yin) [reply src/foraging_gui/Foraging.py]: Sure! Done!

**Comment by alexpiet** [comment]
Can you be more specific about this: "Pay attention to the change of water suggestion"

**Comment by alexpiet** [comment]
Can you make the name of the PR more descriptive?

**Comment by XX-Yin** [comment]
> Lets resolves this issue before merging: [AllenNeuralDynamics/aind-behavior-blog#116](https://github.com/AllenNeuralDynamics/aind-behavior-blog/issues/116)

I fixed this issue. The total water in the information panel only includes the manual water (it should also include actively earned water).  This issue is only related to today's update.

**Pull request merged**